### ISSUE TITLE
Music not playing even after updating to 0.5.13

### ISSUE THREAD

**Issue by vedantanna** [Malopieds/InnerTune#448]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/Malopieds/InnerTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Play any song

### Expected behavior

Song should play 

### Actual behavior

Song not playing. Showing error 

### Screenshots/Screen recordings

![Screenshot_2024-12-26-19-35-56-800_com malopieds innertune](https://github.com/user-attachments/assets/03aa49a5-d76f-42ec-b38d-78e2a4b7b768)

### Logs

Don't know 

### InnerTune version

0.5.13

### Android version

14

**Comment by ALPHAX28** [comment]
Uninstall the app and then reinstall...after reinstallation...do not log in...just start to play the music directly....just confirm if its working...and then close this

**Comment by QwQ3094** [comment]
Though I dont log in, it cant work correctly.

**Comment by Richard38907** [comment]
Not working even after a clean install 🫤

**Comment by ALPHAX28** [comment]
I tested it without logging in and its working perfectly for me..after logging in its showing the unknown error issue...but without logging in...its working perfectly

**Comment by ALPHAX28** [comment]
I guess the issue is with yt servers or some api changes which is causing this issue...i think its not properly fetching the songs and thats why it is causing the unknown error issue

**Comment by frenchy82** [comment]
Innertune had a strange behaviour this morning. While it was working perfectly well since 0.5.13, this morning after the first song it stopped working with the same error as the previous version. It could no longer find the requested songs. It's like it got lost.
After stopping Innertune and starting a new song search, it's working perfectly again for the moment

**Comment by CranialTyrant** [comment]
Yeah, signing into Google seems to break it. Signing out seems to make it work fine again.

**Comment by Richard38907** [comment]
I am not signed in. Just a clean install and I get the error message🤔.

**Comment by ALPHAX28** [comment]
So has anyone figured out where is the issue actually...is is from the yt servers or api or it is an app issue?

**Comment by ecv80** [comment]
After playing several songs it just stopped with unknown error again and refuses to play any more songs. The songs were not cached so I have no idea what's going on. I never logged in since I installed this version.

Edit: to expand on this.
So what happened is it stopped playing with unknown error before it could get to play Five Hours - Deorro
https://music.youtube.com/watch?v=yCwzA8XNBrg

Then when I skipped to the next one it wouldn't play anymore. But just now I have figured that it will play the next songs if I pause and play again. So the problem in my case seems to be with that song alone (for now).

**Comment by Stokesx7** [comment]
> Not working even after a clean install 🫤

After my clean install, I checked in settings and Content. It showed me as still logged in. Touch that and it will take you to the account you're logged in with. Hit your profile pic and select sign out. Go back to inner tune and play music. Works perfectly now.

**Comment by frenchy82** [comment]
I still have, after a long time of listening (30 or 40 minutes for example) the error "unknown error" that comes back. And whatever the song, it doesn't work anymore.
You then have to quit innertune, close it completely and restart it. Then, it works again

Could there be a cache problem or something like that?

**Comment by Lorestan00** [comment]
Has this gone away for anyone after 0.513? It's still happening for me

**Comment by derei** [comment]
Same here. It stops with an "unknown error" just few seconds into a song. Not happening with all songs though. But it will happen consistently with same songs).

**Comment by kyubyy** [comment]
I'm on 0.5.14 now and have now been getting this error today. Has anybody found a fix?
